packages/mesos-modules/buildinfo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "requires": ["mesos"],
+    "docker": "dcos-builder",
+    "single_source" : {
+      "kind": "git",
+      "git": "https://github.com/joerg84/mesos-remove-dockercfg-modules2.git",
```

Comment:
just adding a note here, this needs to move to `dcos/dcos-mesos-modules` pre-merge.